e2e/tests/workspace/workspace-ui.spec.ts

```diff
@@ -0,0 +1,105 @@
+import Workspaces from '../../app/Workspaces';
+import Home from '../../pages/home';
+import { getProperty } from '../../services/element-handler';
+const Chrome = require('../../browser/ChromeBrowser');
+
+jest.setTimeout(60 * 1000);
+
+describe('Workspace-editing page', () => {
+
+  let page;
+
+  beforeEach(async () => {
+    page = await Chrome.setup();
+  });
+
+  afterEach(async () => {
+    await Chrome.teardown();
+  });
+
+  test('Can Create new workspace from Home page', async () => {
+    const home = new Home(page);
+    const link = await home.getCreateNewWorkspaceLink();
+    expect(link).toBeTruthy();
+    await link.click();
+
+    // check page title
+    const actPageTitle = await page.title();
+    expect(actPageTitle).toMatch('Create Workspace');
+
+    const workspace = new Workspaces(page);
+    // check Workspace-Name textfield
+    expect(await workspace.input_workspace_name()).toBeTruthy();
+    // check dataset select options
+    expect(await workspace.select_dataSet()).toBeTruthy();
+  }, 60 * 1000);
+
+
+  test('Create new workspace page', async () => {
+    const workspaces = new Workspaces(page);
+    await workspaces.goURL();
+    await workspaces.click_button_CreateNewWorkspace();
+
+    // check page title
+    const actPageTitle = await page.title();
+    expect(actPageTitle).toMatch('Create Workspace');
+
+    const workspace = new Workspaces(page);
+
+    // webelement Disease-focused research: out-of-box
+    const elementDiseaseName = await workspace.element_diseaseName();
+    const diseaseNameCheckbox = await elementDiseaseName.checkbox();
+    expect(diseaseNameCheckbox).toBeTruthy();
+    expect(await getProperty(page, diseaseNameCheckbox, 'checked')).toBeFalsy();
+    expect(await getProperty(page, diseaseNameCheckbox, 'disabled')).toBeFalsy();
+
+    const diseaseNameTextInput = await elementDiseaseName.textfield();
+    expect(diseaseNameTextInput).toBeTruthy();
+    expect(await getProperty(page, diseaseNameTextInput, 'disabled')).toBeTruthy();
+
+    // await (diseaseNameCheckbox).click(); // click checkbox to change
```

Comment:
how do these 2 lines differ in approach?

Reply:
commented Puppeteer `.click()` function failed to click checkbox. It could be an issue in Puppeteer tool. But I don't really understand why. `click` by JS native function worked.